app/google-services.json

```diff
@@ -1,25 +1,25 @@
 {
   "project_info": {
     "project_number": "1041583037627",
-    "firebase_url": "https://traceit-ae280.firebaseio.com",
+    "firebase_url": "https://traceitfirebase login-ae280.firebaseio.com",
```

Comment:
URL is not valid. Is it just for reference here?

Reply:
Yes this was intensional. The file is just here to make sure CI pipeline passes. Most likely it'll be best to keep the file securely on github.

Reply:
Fixed in later commits.